velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -48,10 +55,20 @@ class NthValueTest : public WindowTestBase {
     // be outside the partition also. The error cases for -ve offset values
     // are tested separately.
     testWindowFunction(input, "nth_value(c3, c2)");
+
+    // The first_value, last_value functions are tested for columns c0, c1, and
+    // c2, which contain data with different distributions.
+    testWindowFunction(input, "first_value(c0)");
```

Comment:
I'm seeing this test taking a very long time, > 5 min. Would it be possible to shorten it or break it down into multiple shorter tests? CC: @aditi-pandit